sequencer/README.md

```diff
@@ -19,5 +21,18 @@ task test-<name> # for example: task test-historydb
 ### Sync flow
 <img src="../doc/images/sequencer_sync_flow.png" />
 
+<b>CMD to run sequencer in sync mode</b>
```

Comment:
How about using markdown headers instead of html tags? Github generates table of contents using markdown headers and they are also good for SEO. So something like `## CMD to run sequencer in sync mode`